explanations/dispatchers.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>A short explainer why we expect these particular results would be great. You don't want to go through the functions to understand what is happening when reading this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/lcm/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>